package.json

```diff
@@ -8,7 +9,9 @@
     "preview": "nuxt preview"
   },
   "dependencies": {
-    "@nuxt/content": "^2.13.4",
-    "nuxt": "^3.15.2"
+    "@nuxt/content": "next",
```

Comment:
better if we specify an exact version so renovate will force the update (without this, stackblitz will get stuck on an old version, for example)